.vscode/tasks.json

```diff
@@ -1,41 +0,0 @@
-// Available variables which can be used inside of strings.
-// ${workspaceRoot}: the root folder of the team
-// ${file}: the current opened file
-// ${relativeFile}: the current opened file relative to workspaceRoot
-// ${fileBasename}: the current opened file's basename
-// ${fileDirname}: the current opened file's dirname
-// ${fileExtname}: the current opened file's extension
-// ${cwd}: the current working directory of the spawned process
-{
-    // See https://go.microsoft.com/fwlink/?LinkId=733558
-    // for the documentation about the tasks.json format
-	"version": "2.0.0",
-
-    // Start PowerShell
-    "windows": {
-        "command": "${SystemRoot}/Program Files/PowerShell/7/pwsh.exe",
-        "args": [ "-NoProfile", "-ExecutionPolicy", "Bypass" ]
-    },
-    "linux": {
-        "command": "/usr/bin/powershell",
-        "args": [ "-NoProfile" ]
-    },
-    "osx": {
-        "command": "/usr/local/bin/powershell",
-        "args": [ "-NoProfile" ]
-    },
-
-    // Associate with test task runner
-    "tasks": [
-        {
-            "taskName": "Test",
-            "suppressTaskName": true,
-            "isTestCommand": true,
-            "args": [
-                "Write-Host 'Invoking Pester...'; $ProgressPreference = 'SilentlyContinue'; Invoke-Pester -Script test -PesterOption @{IncludeVSCodeMarker=$true};",
```

Comment:
If this VSCode file is going to be removed, the invoke line should be added to the documentation.
 ```
Invoke-Pester -Script test -PesterOption @{IncludeVSCodeMarker=$true};
```

Reply:
Nevermind, the command is in the workflows here:

https://github.com/techservicesillinois/SecOps-Powershell-CISDSC/blob/736d0b18f143982c7b65c5d9b6f9215cb57391a3/.github/workflows/pester_CISDSC.yml#L23-L28